docs/source/science/NISAR/OPERA Surface Displacement.ipynb

```diff
@@ -0,0 +1,4440 @@
+{
```

Comment:
<p>Change "In this tutorial..." to "This tutorial..."</p><p> </p><p>Also, add a period at the end of the second sentence.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>